src/instr.cpp

```diff
@@ -356,7 +360,12 @@ int main(int argc, char *argv[]) {
 	// Get module from LLVM file
 	LLVMContext &Context = getGlobalContext();
     SMDiagnostic Err;
-    Module* m = ParseIRFile(argv[2], Err, Context);
+#if (LLVM_VERSION_MINOR >= 8)
```

Comment:
The same with [pP]arseIRFile (LLVM 3.5). I'd be better to make it compile even on LLVM 3.5 - 3.7, since you're version of KLEE can use these versions.